src/openforms/appointments/contrib/jcc/apps.py

```diff
@@ -0,0 +1,9 @@
+from django.apps import AppConfig
+
+
+class JCCAfsprakenPlugin(AppConfig):
+    name = "openforms.appointments.contrib.jcc"
+    verbose_name = "JCC-Afspraken appointment plugin"
```

Comment:
```suggestion
    verbose_name = "JCC-Afspraken appointment plugin"
```
Please mark this translatable